idunn/datasources/hove/client.py

```diff
@@ -10,6 +10,7 @@
 DIRECT_PATH_MAX_DURATION = 86400  # 24h
 MIN_NB_JOURNEYS = 2
 MAX_NB_JOURNEYS = 5
+FREE_RADIUS = 50  # meters
```

Comment:
I try to understand https://doc.navitia.io/?shell#journeys : we don't calculate the time between the given coordinate and the journey departure/arrival in a radius of 50m ?

Reply:
Yeah exactly, in my understanding it will help with not having weird shenanigans where the itinerary will go around the starting building or take a weird exit from the arrival.

It will affect the computed duration so it shouldn't be more than 50m, 50m meters might already be too much as it means we consider that the user can teleport for a distance of 100m overall.